### PR TITLE
Alpha/input styling

### PR DESCRIPTION
**Pull request by iondrimba**
**CHANGELOG** :memo:

* Updating form input styles
* Updated documentjs default layout
* Added outlined styles for radios and checkboxes

captura de tela 2017-04-24 as 7 22 50 pm

### CODE REVIEW

File: src/scss/components/_form.scss [hunk 1]
```diff
@@ -59,18 +115,21 @@
...
     margin-bottom: 6px;
     color: get-color((color: blackNeutral, opacity: .9));
-    font-size: 15px;
+    font-size: 13px;
     font-weight: 700;
   }
 
   &__label-link {
     margin-left: 10px;
+    float: right;
```
Comment (eduardojmatos) [inline src/scss/components/_form.scss]: I ran the project here, and with a large label, this `form__label-link` seems break, like this sample:

screen shot 2017-04-24 at 9 26 56 pm

In this case, do you think that `display: block` and `text-align: right` will work better? See the sample above:

screen shot 2017-04-24 at 9 31 34 pm

I think that is a decision to designer...

Comment (iondrimba) [inline src/scss/components/_form.scss]: @eduardojmatos Nice solution with the text-align:right. I completely forgot about that option :)

File: theme/templates/layout.mustache [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+			<div class="logo-menu">
+				<a class="brand" href="./index.html">{{makeParentTitle}}</a>
+			</div>
+
+			<div class="pull-right">
+				<div class="bitovi-menu">
+					<a href="http://bitovi.com" class="bitovi icon-bits">Bitovi</a>
+					<ul class="dropdown-menu">
+						<li><a href="http://bitovi.com">Bitovi.com</a></li>
```
Comment (iondrimba) [inline theme/templates/layout.mustache]: @eduardojmatos Yes we can, future plans would be to customize everything :)

File: src/scss/components/_form.scss (hunk 1 above)
Comment (iondrimba) [inline src/scss/components/_form.scss]: @eduardojmatos I talked to the designer and he said it is ok to keep the label like the first example.

**Comment by iondrimba** [comment]
@eduardojmatos adicionado radios e checkboxes com estile outline.
captura de tela 2017-04-25 as 1 05 30 pm

File: theme/static/styles/layout.less [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+		}
+		&:hover > a {
+			background-color: rgba(0, 0, 0, 0.1);
+			border: 1px solid rgba(0, 0, 0, 0.3);
+			border-radius: 2px;
+		}
+		&.active > a {
+			background-color: rgba(0, 0, 0, 0.35);
+			border: 1px solid rgba(0, 0, 0, 0.6);
+			border-radius: 2px;
```
Comment (MarcoBrunoBR) [inline theme/static/styles/layout.less]: go go go variable?

Comment (iondrimba) [inline theme/static/styles/layout.less]: @MarcoBrunoBR Yes, this is a new border-radius :)

Comment (iondrimba) [inline theme/static/styles/layout.less]: @MarcoBrunoBR this is not a gaiden file, it's the documentation template style. Disconsider the variable creation

**Comment by iondrimba** [comment]
@MarcoBrunoBR @eduardojmatos Anything that still needs to change?

File: src/scss/components/form/_radio.scss [hunk 4]
```diff
@@ -62,4 +67,41 @@
       content: '';
     }
   }
+
+  &--disabled {
+    label {
+      opacity: .5;
+    }
+    label::before {
```
Comment (eduardojmatos) [inline src/scss/components/form/_radio.scss]: This can be on label scope

**Comment by iondrimba** [comment]
@eduardojmatos changes updated

**Pull request merged**